### PR TITLE
Update Ui class puml and DG

### LINKED ISSUE #162: Update Ui class puml and DG

### PR DESCRIPTION
**Pull request by NorbertLoh**
Fixes #162.

### CODE REVIEW

**Comment by NorbertLoh** [comment]
Heres what it looks like!
image

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -29,16 +29,13 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
...
 
 Given below is a quick overview of main components and how they interact with each other.
 
 **Main components of the architecture**
 
-**`Main`** (consisting of
-classes [`Main`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/Main.java)
-and [`MainApp`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/MainApp.java)) is
-in charge of the app launch and shut down.
+**`Main`** (consisting of classes [`Main`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/Main.java) and [`MainApp`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/MainApp.java)) is in charge of the app launch and shut down.
```
Comment (Ma-Xizi) [inline docs/DeveloperGuide.md:38]: Should we change it to the main.java in our file instead?

Comment (NorbertLoh) [reply docs/DeveloperGuide.md:38]: I think it should be Main.java the file name is Main.java.

Comment (NorbertLoh) [reply docs/DeveloperGuide.md]: Ok I have updated it.

**Review by ethantyh692 — APPROVED** [review]
LGTM

**Pull request merged**